Cython/Utility/ModuleSetupCode.c

```diff
@@ -698,6 +698,10 @@ class __Pyx_FakeReference {
   #define __Pyx_PyCode_HasFreeVars(co)  (PyCode_GetNumFree(co) > 0)
   #define __Pyx_PyFrame_SetLineNumber(frame, lineno)
 #else
+#if PY_VERSION_HEX >= 0x030b00A5
+  #include "frameobject.h"
+  #include "internal/pycore_frame.h"
+#endif
```

Comment:
I'd remove this because we only define macros here. Whether they are being used or not depends on other code elsewhere, and that should then do the includes (and does it already).
```suggestion
```